test-utils/state-viewer/src/main.rs

```diff
@@ -44,9 +42,11 @@ fn kv_to_state_record(key: Vec<u8>, value: DBValue) -> StateRecord {
             if separator.is_some() {
                 StateRecord::Data { key: to_base64(&key), value: to_base64(&value) }
             } else {
-                let proto: account_proto::Account = parse_from_bytes(&value).unwrap();
-                let account: Account = proto.try_into().unwrap();
-                StateRecord::Account { account_id: to_printable(&key[1..]), account }
+                let account = Account::from_slice(&value).unwrap();
+                StateRecord::Account {
+                    account_id: to_printable(&key[1..]),
```

Comment:
converting bytes to string using `to_printable` is a scam, since if we increase account length to 64, this will break